index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>cache validation 1</title>
+
+  <!-- Bower Dependencies -->
+  <script src="bower_components/widget-common/dist/rise-cache.js"></script>
```

Comment:
@pcsandford Your import of `rise-cache.js` would be better loaded not in `<head>` but just after jquery in `<body>`.